resources/views/profile/partials/update-profile-information-form.blade.php

```diff
@@ -145,10 +145,28 @@ class="mt-2"
             />
         </div>
 
+        <div>
+            <x-input-label
+                for="prefers_questions"
+                :value="__('Would you like to receive questions?')"
+            />
+            <x-select-input
+                name="prefers_questions"
+                class="mt-1 block w-full"
+                class="mt-1 block w-full"
+                :options="[true => __('People can ask me questions'), false => __('I do not want to receive questions')]"
+                :value="old('prefers_questions', $user->prefers_questions)"
+            />
+            <x-input-error
+                class="mt-2"
+                :messages="$errors->get('prefers_questions')"
+            />
+        </div>
+
         <div>
             <x-input-label
                 for="prefers_anonymous_questions"
-                :value="__('How would you like to do questions by default?')"
+                :value="__('How would you like to do questions by default? Will only work when you accept questions.')"
```

Comment:
why?

Reply:
prefers_anonymoud_question applicable when you ask questions.

So both are not related right now!

Reply:
Oh okay, I thought they are related. Thanks for clarifying. I will reverse that line.